addons/web/static/src/core/tooltip/tooltip_service.js

```diff
@@ -86,7 +86,9 @@ export const tooltipService = {
                 positionY < targetRect.top ||
                 positionY > targetRect.bottom
             ) {
-                return true; // mouse is no longer hovering the target
+                // mouse is no longer hovering the target
+                // cleanup inside function to reset timeout and avoid false detection on top/left mouseenter (chromium issue)
+                cleanup();
```

Comment:
Not sure about this, we'll see during the tech review

Reply:
I doubt this could solve any issue. Maybe it's a bug in a specific chrome version?

Reply:
@aab-odoo it is specific to chromium yes, it is a bug. I can't identify why and I agree that the fix seems to not change anything but it does make it work. Just tested on safari Version 16.1 (18614.2.9.1.12) and chrome Version 109.0.5414.87 (Official Build) (arm64). The entry from top and left is not showing the tooltip without this fix.

Reply:
I'm on chrome 108 and I can't reproduce the issue

Reply:
Weird, could it be because I'm on MacOS?

Reply:
I can confirm the issue on Chrome 108 (OSX)


https://user-images.githubusercontent.com/8558351/213167056-3bd84b3d-446d-4056-bbfb-7eaeb1899c4a.mov

Reply:
@aab-odoo I tested with my colleagues from RD-Design, I confirm the issue is there on MacOS Ventura and Monterey (tested with chrome 108, 109 and Safari 16). Executing the `cleanup()` in the `shouldCleanUp()` provides the expected behavior.